src/controllers/controllermanager.h

```diff
@@ -85,7 +85,7 @@ class ControllerManager : public QObject {
     void stopPolling();
     void maybeStartOrStopPolling();
 
-    static QString presetFilenameFromName(QString name) {
+    static QString sanitizeString(QString name) {
         return name.replace(" ", "_").replace("/", "_").replace("\\", "_");
```

Comment:
It is a shame that Qt does not provide a solution for this common problem. I was just dealing with this the other day for #2618. Could you move this to `src/util/string.h`? Also rename the function to `sanitizeFileName`.

Reply:
I don't think this needs to replace spaces, does it? What does not support spaces in file names?

Reply:
> I don't think this needs to replace spaces, does it? What does not support spaces in file names?

That was the main reason I renamed it. It's sometimes used for file names, but it's also uses for formatting the device name so that it can be used as a config key. I'll rename the function to `sanitizeDeviceName`.

Reply:
okay